code_samples/tool_window/README.md

```diff
@@ -0,0 +1,29 @@
+# Tool Window Sample [![JetBrains IntelliJ Platform SDK Docs](https://jb.gg/badges/docs.svg)][docs]
+*Reference: [Tool Windows in IntelliJ SDK Docs][docs:tool_windows]*
+
+## Quickstart
+
+Tool Windows are child windows of the IDE used to display information. These windows generally have their own toolbars
+(referred to as tool window bars) along the outer edges of the main window containing one or more tool window buttons,
+which activate panels displayed on the left, bottom, and right sides of the main IDE window.
+
+Current implementation displays a `JPanel` component containing simple icons and information regarding the actual
```

Comment:
Suggest: _The current implementation displays a `JPanel` component containing simple icons and information about the actual..._